src/code_8006E9C0.c

```diff
@@ -172,7 +173,7 @@ void func_8006EEE8(s32 courseId) {
 // I really really don't know what D_8018D9B4 is supposed to be
 // Its value is a pointer that seems to point to compressesed data
 // of some variety (sometimes TKMK, sometimes MIO0)
-extern u8 *D_8018D9B4;
+extern uintptr_t *D_8018D9B4;
```

Comment:
Usually the type here should be what the data points at.

I think u8 is more accurate. But I'm not super concerned either.

Reply:
with u8 match